notification-service/src/test/resources/configs/notification-service/application.conf

```diff
@@ -8,6 +8,22 @@ logger.file.dir = "/var/logs/anomaly-event-processor"
 
 notification.channels.path = "build/resources/test/configs/rules.json"
 
+ruleSource = {
```

Comment:
can you make this change in the main config file and `hypertrace-alert-engine` module as well